vmdb/app/models/ems_microsoft.rb

```diff
@@ -25,9 +25,9 @@ def self.raw_connect(username, password, auth_url)
     WinRM::WinRMWebService.new(auth_url, :ssl, :user => username, :pass => password, :disable_sspi => true)
   end
 
-  def self.auth_url(ipaddress, port = nil)
+  def self.auth_url(hostname, port = nil)
     port ||= 5985
-    "http://#{ipaddress}:#{port}/wsman"
+    "http://#{hostname}:#{port}/wsman"
```

Comment:
@jrafanie This looks like it needs a fix for IPv6 (i.e URI building)...I assume you're going to do it in another PR?